src/main/java/io/jenkins/plugins/appcenter/task/internal/UploadAppToResourceTask.java

```diff
@@ -84,11 +91,54 @@
         final String pathToDebugSymbols = request.pathToDebugSymbols;
         final String symbolUploadUrl = requireNonNull(request.symbolUploadUrl, "symbolUploadUrl cannot be null");
 
-        log("Uploading symbols to resource.");
+        final File file = new File(filePath.child(pathToDebugSymbols).getRemote());
+        if (file.length() > (1024 * 1024) * 256) {
+            return uploadSymbolsChunked(request, symbolUploadUrl, file);
+        } else {
+            return uploadSymbolsComplete(request, symbolUploadUrl, file);
+        }
+    }
+
+    @Nonnull
+    private CompletableFuture<UploadRequest> uploadSymbolsChunked(@Nonnull UploadRequest request, @Nonnull String symbolUploadUrl, @Nonnull File file) {
+        log("Uploading symbols to resource chunked.");
 
         final CompletableFuture<UploadRequest> future = new CompletableFuture<>();
+        CompletableFuture.supplyAsync(() ->
+        {
```

Comment:
Just a small formatting change here........ can you move the `{` to the line above after the `->`